README.md

```diff
@@ -151,6 +152,76 @@ Select a new parameter file or create a new parameter file to update the associa
 
 ![Image showing the parameter file creation options (none, new, and browse)](./images/undo-change-mapping-two.png)
 
+## Linked Template Support
+
+When a [linked template](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/linked-templates#linked-template) is referenced, schema and parameter validations are provided.
+
+Linked template support requires that all top-level parameters have a value. This means that a parameter file is associated with the template OR all top-level parameter definitions have a default value.
+
+Scenarios supported:
+
+- Relative path (requires apiVersion 2020-10-01 or higher of Microsoft.Resources/deployments):
+
+```json
+        {
+            "name": "linkedDeployment1",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    // Relative to current template's folder
+                    "relativePath": "child.json"
+                },
+                "parameters": {
+                }
+            }
+        }
+```
+
+- Full URI:
+
+```json
+        {
+            "name": "linkedDeployment1",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-10-01",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/StephenWeatherford/template-examples/master/linkedTemplates/uri/child.json"
```

Comment:
Let's find a link from the quick starts repo (yes, I'll update the changelog too).
Also, I think it should use expressions.  Perhaps the _artifactionsLocation pattern.